packages/core-contracts/test/contracts/token/ERC721.test.js

```diff
@@ -0,0 +1,513 @@
+const { ethers } = hre;
+const assert = require('assert/strict');
+const assertRevert = require('@synthetixio/core-js/utils/assert-revert');
+const assertBn = require('@synthetixio/core-js/utils/assert-bignumber');
+const { findEvent } = require('@synthetixio/core-js/utils/events');
+
+describe('ERC721', () => {
```

Comment:
massive test... if you think/consider a way to reduce it or to split it, more than welcome to do it